src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -830,6 +831,38 @@ public function isAbstract()
     }
 
     /**
+     * Whether this definition is deprecated, that means it should not be called
+     * anymore.
+     *
+     * @param bool $boolean
+     *
+     * @return Definition the current instance
+     *
+     * @todo Support a suggested definition instead ?
+     *
+     * @api
+     */
+    public function setDeprecated($boolean = true)
```

Comment:
I would prefer using $deprecated instead of $boolean as variable and the default value here doesn't make sense.